src/Components/Components/src/OwningComponentBase.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable disable warnings
```

Comment:
Why?

Reply:
I had thought updating the implementation might have been a pain, but I should just do this.